.github/silabs-builds-mg26.json

```diff
@@ -3,6 +3,10 @@
         {
             "boards": ["BRD4116A", "BRD4117A", "BRD4118A", "BRD2608A"],
             "arguments": ["--docker"]
+        },
+        {
+            "boards": ["BRD2608A"],
+            "arguments": ["--docker", "--enable_tracing", "--clean"]
         }
```

Comment:
This is going to run for all samples. Should we run it only for the ligthing app instead?